scripts/browse.py

```diff
@@ -11,6 +11,10 @@ def scrape_text(url):
     if not url.startswith('http'):
         return "Error: Invalid URL"
     
+    # Restrict access to local files
+    if url.startswith('file://') or url.startswith('file://localhost'):
```

Comment:
The second condition is redundant. Or did you mean `http(s)://localhost`?

Reply:
I meant to write `file:///` for the first condition. My mistake.